Prace_domowe/Praca_domowa3/kinga_ulasik/PD3_Kinga_Ulasik.ipynb

```diff
@@ -0,0 +1,1223 @@
+{
```

Comment:
<p>Tu na pewno wszystko jest dobrze? Trochę to zbyt liniowe, prawdaż?</p><p> </p><p>Na pewno coś jest nie tak - zobacz plot_roc_cruve</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>